### ISSUE TITLE
GIF picker crashes frequently in 0.9.17 on Windows.

### ISSUE THREAD

**Issue by shakesoda** [telegramdesktop/tdesktop#1467]
I can't tell if it's related to specific ones or not, but when loading or scrolling more into view it crashes the client (sometimes as soon as I open it, sometimes I have to scroll down to load a few more - but it always happens).

**Comment by telegramdesktop** [comment]
@shakesoda Please contact me at telegram.me/preston and send the crash dumps from "tdumps" folder near Telegram.exe file.

**Comment by shakesoda** [comment]
it doesn't seem to be producing any crash dumps for me, do I need to put it in some sort of debug mode or use a debug binary?

**Comment by telegramdesktop** [comment]
@shakesoda No, each crash on Windows should produce those :( can you download the portable version from https://desktop.telegram.org, extract it to some temp folder, launch it from there, log in and crash — and see, if "tdumps" folder is created near the "tdata" folder inside this temp folder you chose.

**Comment by shakesoda** [comment]
Update since it's been a few more hours and I've crashed it a lot of times:

* It's not caused by specific files, but it is caused by ones I've uploaded from the same client
* Often I'm able to access the GIF list and send something once, but when I try to use it again it's very likely to crash.
* Errors show up in the log, but I've only managed to get one dump (which I've sent). The files that show up in the log work fine.

Log errors:
```
[2016.01.04 20:42:51] Gif Error: Unable to avcodec_decode_video2() for file 'path/to/some.gif', data size '1139883', error -1094995529, Invalid data found when processing input
[2016.01.04 20:42:52] Gif Error: Unable to avcodec_open2 for file 'path/to/some.gif', data size '1628025', error -22, Invalid argument
```

**Comment by telegramdesktop** [comment]
Please check, it should be fixed in the last 0.9.18 stable version.
